src/Fahrplanauskunft/Fahrplanauskunft/Objekte/FahrplanauskunftSpeicher.cs

```diff
@@ -79,15 +95,55 @@ public void Laden()
         }
 
         /// <summary>
+        /// Speichert das Objekt Linie aus dem FahrplanauskunftSpeicher in den Ordner
```

Comment:
Kommentar anpassen: Es sind Objekte, also Mehrzahl.
z.B.
Speichert die Objekte Linie aus dem FahrplanauskunftSpeicher in den Ordner